src/frontend/src/app/header/settings/settings.component.ts

```diff
@@ -12,17 +10,12 @@ import { ModeType } from '../modes/modeType.enum';
 })
 export class SettingsComponent implements OnInit {
   settings!: Settings;
-  mode!: Mode;
   modeType = ModeType;
```

Comment:
modeType wordt niet gebruikt, ook ipv instance variabele kan je net zo goed ModeType gebruiken.

Reply:
Instance?

Reply:
Its being used in the HTML